test/cli/import-jdl.spec.js

```diff
@@ -391,18 +391,12 @@ describe('JHipster generator import jdl', () => {
             assert.file([
                 path.join('docker-compose', '.yo-rc.json'),
                 path.join('kubernetes', '.yo-rc.json'),
-                path.join('openshift', '.yo-rc.json'),
-                path.join('rancher-compose', '.yo-rc.json')
+                path.join('openshift', '.yo-rc.json')
             ]);
         });
         it('calls deployment generator', () => {
             expect(subGenCallParams.count).to.equal(4);
```

Comment:
As you deleted 1 subgen, you need to apply this diff. At least, it works for me:

```diff
diff --git a/test/cli/import-jdl.spec.js b/test/cli/import-jdl.spec.js
index eafac03ea..b1620f065 100644
--- a/test/cli/import-jdl.spec.js
+++ b/test/cli/import-jdl.spec.js
@@ -383,7 +383,7 @@ describe('JHipster generator import jdl', () => {
         beforeEach(done => {
             testInTempDir(dir => {
                 fse.copySync(path.join(__dirname, '../templates/import-jdl'), dir);
-                importJdl(['deployments.jdl'], { skipInstall: true }, env, mockFork(done, 4));
+                importJdl(['deployments.jdl'], { skipInstall: true }, env, mockFork(done, 3));
             });
         });
 
@@ -395,7 +395,7 @@ describe('JHipster generator import jdl', () => {
             ]);
         });
         it('calls deployment generator', () => {
-            expect(subGenCallParams.count).to.equal(4);
+            expect(subGenCallParams.count).to.equal(3);
             expect(subGenCallParams.commands).to.eql(['jhipster:docker-compose', 'jhipster:kubernetes', 'jhipster:openshift']);
             expect(subGenCallParams.options[0]).to.eql(['--skip-prompts', '--skip-install', '--force', '--from-cli']);
         });
```

Reply:
Wow thank you @pascalgrimaud you rock !

Reply:
@PierreBesson Please add some comments to this line, otherwise the next time we add/remove subgens it will be dificult to found this needed changes ;)